flake.nix

```diff
@@ -15,38 +15,91 @@
         flake-utils.follows = "flake-utils";
       };
     };
+    crate2nix = {
+      url = "github:nix-community/crate2nix";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
   };
 
-  outputs = { self, nixpkgs, rust-overlay, flake-utils, solc }:
+  outputs = { self, nixpkgs, rust-overlay, flake-utils, solc, crate2nix }:
     flake-utils.lib.eachDefaultSystem (system:
       let
         pkgs = import nixpkgs {
           inherit system;
-          overlays = [ rust-overlay.overlays.default solc.overlay ];
+          overlays = [
+            rust-overlay.overlays.default
+            solc.overlay
+            (final: prev: {
+              solc = (solc.mkDefault final final.solc_0_8_27);
```

Comment:
solc `0.8.28` has been released: https://soliditylang.org/blog/2024/10/09/solidity-0.8.28-release-announcement

Reply:
The source we retrieve it from is https://github.com/alloy-rs/svm-rs/blob/b97a212ddd72671d3f93d3b8c773ffe2edfb1ece/crates/svm-rs/src/install.rs#L251